README.md

```diff
@@ -0,0 +1,105 @@
+# Change-impact-analysis Tool
+## Table of Content
+- [Introduction](#introduction)
+- [Innovation](#innovation)
+- [How to Use](#how-to-use)
+- [Files Generated](#files-generated)
+- [Structure of the Tool](#structure-of-the-tool)
+  - [I. Compilation File List Generation](#i-compilation-file-list-generation)
+  - [II. Git Diff Report Generation](#ii-git-diff-report-generation)
+  - [III. Commit Metadata Retrieval](#iii-commit-metadata-retrieval)
+  - [IV. Web Script Generation](#iv-web-script-generation)
+
+## Introduction
+
+The Change Impact Analysis Tool generates a comprehensive visual report detailing changes in both header files and source code between two Linux versions (tags in the Linux kernel repository: old_tag and new_tag). This tool helps developers view updates from the old version.
```

Comment:
Since we created a new name, do you want to say something like "DeltaKernel Change Impact Analysis Tool"?

Reply:
Have renamed it as well as other similar cases.